cpp/cppsdk/gsdk.cpp

```diff
@@ -4,22 +4,34 @@
 #include "gsdkCommonPch.h"
 #include "gsdkInternal.h"
 #include "gsdkConfig.h"
+#include <stdio.h>
+#include <assert.h>
 
 namespace Microsoft
 {
     namespace Azure
     {
         namespace Gaming
         {
-            std::unique_ptr<GSDKInternal> GSDKInternal::m_instance = nullptr;
-            std::mutex GSDKInternal::m_gsdkInitMutex;
+            GSDKInternal* GSDKInternal::m_instance = nullptr;
```

Comment:
Converting from a unique_ptr to a raw pointer seems undesirable and unnecessary. All the other changes are compatible with this remaining as a unique_ptr